### PR TITLE
Add fuel pump action

### PR DESCRIPTION
**Pull request by NiiRoZz**
Add fuel pump action with check and you need enable in config_master. Default is false. you can change the price in fn_FuelStatOpen.sqf, Line 22 : life_fuelPrices = 80;

### CODE REVIEW

**Comment by NiiRoZz** [comment]
Ho sorry , daniel will do it i think.

**Comment by danielstuart14** [comment]
if you want I can do the changes.
Just need to hold on a little. I'm finishing some college researches.

**Comment by NiiRoZz** [comment]
@danielstuart14 Yes, please because if i do change i need to put new pull request. And i have send you emall in daniel.stuart14@gmail.com. you have not answered my email if you can it's nice.

File: Altis_Life.Altis/core/fn_Setup_Sation_Service.sqf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+[20784.8,16665.9,-0.0521202],
+[20789.6,16672.3,-0.0213318],
+[23379.4,19799,-0.0544052],
+[25701.2,21372.6,-0.0774155]
+];
+_pumpClass = "Land_fs_feed_F";
+{
+	_pump = (nearestObject [_x, _pumpClass]);
+	_pump setFuelCargo 0;
+	_pump addAction ["Ravitaillez en essence le véhicule le plus proche", life_fnc_FuelStatOpen, 1, 3, true, true, "", ' _this distance _target < 5 && cursorTarget == _target '];
```
Comment (sargken) [inline Altis_Life.Altis/core/fn_Setup_Sation_Service.sqf]: String Table?

File: Altis_Life.Altis/dialog/FuelStat.h [hunk 2]
```diff
@@ -0,0 +1,148 @@
+class Life_FuelStat
```
Comment (sargken) [inline Altis_Life.Altis/dialog/FuelStat.h]: shouldn't file be hpp?

**Pull request merged**